doc/sphinx-guides/source/admin/external-tools.rst

```diff
@@ -46,6 +52,12 @@ To list all the external tools that are available in a Dataverse installation:
 
   curl http://localhost:8080/api/admin/externalTools
 
+This API is Superuser only. Note the endpoint difference (/api/externalTools instead of /api/admin/externalTools).
+
+.. code-block:: bash
+
+  curl -s -H "X-Dataverse-key:$API_TOKEN" http://localhost:8080/api/externalTools
```

Comment:
Obviously, only superusers should be able to ADD tools but have we considered allowing the guest user (no API token) LIST tools? (This would apply to the "list one tool" endpoint as well, of course.)

I don't THINK there would be any harm...

On a related note, it's now possible to list (without an API token), the exporters that are enabled:

- #10739

I think it would be nice to have this for external tools as well. We could, of course, defer this and open up the listing API in a future pull request. I just thought I'd mention it.

Reply:
Great, done in 4faadf6 (thanks!) and no objections [in Slack](https://iqss.slack.com/archives/C010LA04BCG/p1734465182633839):

![Screenshot 2024-12-19 at 10 12 20 AM](https://github.com/user-attachments/assets/b8587fce-1c7c-4c7e-af02-795b427864f7)